bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/discovery/NetatmoModuleDiscoveryService.java

```diff
@@ -195,4 +202,46 @@ private ThingUID findThingUID(String thingType, String thingId) throws IllegalAr
 
         throw new IllegalArgumentException("Unsupported device type discovered : " + thingType);
     }
+
+    private String createWeatherStationName(NAMain station, boolean isFavorite) {
+        StringBuilder nameBuilder = new StringBuilder();
+        nameBuilder.append(localizeType(station.getType()));
+        if(station.getStationName() != null) {
```

Comment:
Please apply the formatting standards set for the project. This is something to setup in your IDE. Additionally you may have a command to reformat a source file but I don't know it. 
As an example, there is a space between the if and it's condition.

Reply:
ok, I have now executed the Maven goal spotless:apply. There were a few other files which had formatting errors. Now spotless:check is also running successfully. :-)